test/run_blackbox_tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+host=$(hostname)
+meshlinkrootpath="/home/${host}/meshlink"
```

Comment:
I don't have a /home/$(hostname) directory on my machines, nor do I have a wlp2s0 interface. We shouldn't hardcode any of this. I suggest that this script should do two steps:

1. Set sensible defaults
2. Load an optional configuration file that can be used to modify variables if needed

First, if you invoke a script, you can get the directory where the script resides using `${0%/*}`. Then this can be converted to an absolute path with the `realpath` command. So to find the base directory of the MeshLink source code, you could do:

    meshlinkrootpath=$(realpath ${0%/*}/..)

Then, the most likely interface to use is probably the one that the default route points to, so:

    ethifname=$(ip route show default | awk '{print $5}')

The path to the LXC containers can be found using:

    lxcpath=$(lxc-config lxc.lxcpath)

I also have a different LXC bridge interface, but I don't know an easy way to get that information, so let's keep that as `lxcbr0`. Then, after setting the variables, do:

    test -f $HOME/.config/meshlink_blackbox.conf && source $HOME/.config/meshlink_blackbox.conf

That sources the file `~.config/meshlink_blackbox.conf` if it exists, and allows one to override variables.